app.go

```diff
@@ -29,15 +29,15 @@ func init() {
 }
 
 // Provision implements caddy.Provisioner
-func (s *SouinApp) Provision(_ caddy.Context) error {
+func (s SouinApp) Provision(_ caddy.Context) error {
```

Comment:
`Provision` typically change the state of their instance (e.g. add logger), which won't persist with a value receiver rather than a pointer receiver.

Reply:
But we're doing nothing here so we can use a value receiver.

Reply:
You could just remove `Provision` then.